### PR TITLE
Ecosystem Projects

### PR DESCRIPTION
**Pull request by techpubswriter**
Added the fields from the Excel sheet and edited the content.

### CODE REVIEW

File: projects/akash.md [hunk 1]
```diff
@@ -2,11 +2,17 @@
...
+logline: "Akash is an open, decentralized marketplace and a deployment platform for cloud compute. Akash is integrating their serverless supercloud infrastructure into Solana to allow users to easily deploy powerful and low cost servers to run nodes or to scale their application."
 cta: "https://akash.network"
 logo: /img/akash.svg
 category: infra
 status: building
+website: https://akash.network/
+twitter: https://twitter.com/akashnet_
+telegram: https://t.me/akashnw
+discord: https://discord.com/invite/DxftX67
+
```
Comment (rmshea) [inline projects/akash.md:14]: can you remove these extra lines?

File: projects/Untitled-1 [hunk 2]
```diff
@@ -0,0 +1,22 @@
+diff --git a/projects/audius.md b/projects/audius.md
+--- projects/audius.md
++++ projects/audius.md
+@@ -6,7 +6,17 @@
```
Comment (rmshea) [inline projects/Untitled-1:4]: can you remove these?

File: projects/Untitled-1 [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ cta: "https://medium.com/audius/audius-selects-solana-to-scale-community-owned-music-streaming-8a868db644eb"
```
Comment (rmshea) [inline projects/Untitled-1:5]: missing a "slug"

File: projects/akash.md [hunk 4]
```diff
@@ -2,11 +2,17 @@
...
+
```
Comment (rmshea) [inline projects/akash.md:15]: can you remove the whitespace?

File: projects/waves.md [hunk 5]
```diff
@@ -2,15 +2,19 @@
...
 category: infra
 status: closed
+website: https://waves.tech/
+twitter: https://twitter.com/wavesprotocol
+telegram: https://t.me/wavesnews
+discord: https://discord.com/invite/cnFmDyA
 ---
 
-We are thrilled to announce the integration of the Solana protocol into Gravity, a decentralized cross-chain and oracle network based on the Waves protocol.
+Waves is a Crypto-platform for asset/custom token issuance, transfer and trading on the Waves blockchain, with centralised order matching and decentralised settlement.
```
Comment (rmshea) [inline projects/waves.md:16]: can you add a comma after transfer? We like Oxford commas :)

**Review by rmshea — CHANGES_REQUESTED** [review]
if you remove the whitespace and delete `Untitled-1`, we'll be good to merge.

File: projects/Untitled-1 (hunk 3 above)
Comment (techpubswriter) [reply projects/Untitled-1:5]: I didnt delete "slug"

**Review by rmshea — APPROVED** [review]

**Pull request merged**